wallet/reference/provider-api.md

```diff
@@ -256,6 +254,58 @@ For example, if you create a subscription using
 [`eth_subscribe`](/wallet/reference/eth_subscribe), each
 subscription update is emitted as a `message` event with a `type` of `eth_subscription`.
 
+## Methods
```

Comment:
```suggestion
### Remove event listeners
```

There is already a different "Methods" subheading on this page. Maybe since this is directly related to events, we can add this subheading instead?